src/lsm/compaction.zig

```diff
@@ -26,70 +27,97 @@ pub fn CompactionType(
 
         const Grid = GridType(Storage);
         const BlockPtr = Grid.BlockPtr;
+        const BlockPtrConst = Grid.BlockPtrConst;
+        const BlockWrite = struct {
+            write: Grid.Write = undefined,
+            block: BlockPtr = undefined,
+            ready: bool = false,
```

Comment:
nit: What do you think of renaming `ready` to `done`? `ready` could mean "ready for more data" or "ready to be written". Or it could be "closed" (i.e. a "closed" block can't receive any more values, and open block can receive values).

Reply:
I understand; I was just suggesting a less ambiguous name for the field.